utils.py

```diff
@@ -332,10 +334,15 @@ def are_datetimes_close(later_datetime, earlier_datetime):
 
 
 def generate_random_string(length):
+    """Generates a random string of the length specified and returns it.
+    Args:
+      length: Length of the string to be generated.
+    """
     return base64.urlsafe_b64encode(os.urandom(length))
 
 
 def generate_new_session_id():
+    """Generates a new session id."""
     return generate_random_string(24)
```

Comment:
Write both Args and Returns for all the functions along with the datatypes.

Reply:
Args are missing here. Why are you not addressing the comments? Please write- Done! in future so that I can review the things better.